contracts/JBController.sol

```diff
@@ -98,6 +105,12 @@ contract JBController is IJBController, JBTerminalUtility, JBOperatable, Reentra
   // --------------------- private stored properties ------------------- //
   //*********************************************************************//
 
+  /** 
+    @notice
+    Max. Token Rate.
+  */
+  uint constant MAX_TOKEN_RATE = 10000;
```

Comment:
uint256 private constant _MAX_TOKEN_RATE

Also the comment should signal the meaning of the constant, not reiterate the name. Refer to other constants in the codebase please.